runtime/service.go

```diff
@@ -647,12 +648,16 @@ func (s *service) buildVMConfiguration(req *proto.CreateVMRequest) (*firecracker
 
 	var driveBuilder firecracker.DrivesBuilder
 	// Create non-stub drives
+	var path = s.config.RootDrive
 	if root := req.RootDrive; root != nil {
-		driveBuilder = firecracker.NewDrivesBuilder(root.PathOnHost)
-	} else {
-		driveBuilder = firecracker.NewDrivesBuilder(s.config.RootDrive)
```

Comment:
Should we remove this constructor if we're not going to use it?

Reply:
I'd like to keep the constructor to be backward-compatible. The SDK is pre 1.0, but keeping the constructor doesn't cost us much.